build.gradle

```diff
@@ -20,7 +20,8 @@ ext {
 
   // Third parties
   offheapVersion = '2.2.2'
-  statisticVersion = '1.1.0'
+  managementVersion = '5.0.0.beta'
```

Comment:
you mean remove line: managementVersion = '5.0.0.beta'?  This was in the code I inherited from Ludovic.